lib/uplink/object.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// ObjectMeta contains metadata about a specific Object
+type ObjectMeta struct {
```

Comment:
Why doesn't it have `ContentType`, `EncryptionScheme` and `RedundancyScheme` too?

Reply:
I thought ContentType could go in Metadata. EncryptionScheme and RedundancyScheme could be added, but i only want to add the bare minimum for this first draft